app/models/scheme.rb

```diff
@@ -166,7 +166,7 @@ def synonyms
   end
 
   def appended_text
-    "#{locations.count { |location| location.startdate.blank? || location.startdate <= Time.zone.today }} locations"
+    "#{confirmed_locations_count} completed #{'location'.pluralize(confirmed_locations_count)}, #{unconfirmed_locations_count} incomplete #{'location'.pluralize(unconfirmed_locations_count)}"
```

Comment:
we use language completed/incomplete on the frontend but actually this is just the same as confirmed/unconfirmed for purposes in the code. It made the most sense to juxtapose these here rather than earlier on e.g. by making a completed scope that was actually just checking for confirmed